### PR TITLE
Add 1.6.24 & 1.6.25 post

### PR DESCRIPTION
**Pull request by Nightfirecat**
Also add a commit to adjust `<video>` styles because they don't look good if they try to overflow the table/blog container.

### CODE REVIEW

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+- The [Inventory Grid plugin](https://github.com/runelite/runelite/wiki/Inventory-Grid) now operates
+  while viewing your bank
+- An issue causing Discord party invites to fail has been fixed
+- Tooltip labels have been added to clue scroll and minigame entries in the [HiScore
+  plugin](https://github.com/runelite/runelite/wiki/HiScore)
+- The [Agility plugin](https://github.com/runelite/runelite/wiki/Agility) now estimates your laps
+  per hour more accurately
+- Some Godwars Dungeon obstacles and the Prifddinas portals are now highlighted properly by the
+  [Agility plugin](https://github.com/runelite/runelite/wiki/Agility)
+- The [Clue Scroll plugin](https://github.com/runelite/runelite/wiki/Clue-Scroll) plugin now shows a
```
Comment (Alexsuperfly) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:58]:
```suggestion
- The [Clue Scroll plugin](https://github.com/runelite/runelite/wiki/Clue-Scroll) now shows a
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  location description for map clues
+- The [Cannon plugin](https://github.com/runelite/runelite/wiki/Cannon) now suggests a spot for
+  Meiyerditch Laboratory Bloodvelds
+- You can now look up your Nightmare kill count with the [Chat Commands
+  plugin](https://github.com/runelite/runelite/wiki/Chat-Commands) using the `nm`, `tnm`, and
+  `nmare` abbreviations
+- Teleport tablets are now labeled by the [Item Identification
+  plugin](https://github.com/runelite/runelite/wiki/Item-Identification)
+- The [Nightmare Zone plugin](https://github.com/runelite/runelite/wiki/Nightmare-Zone) plugin can
```
Comment (Alexsuperfly) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:67]:
```suggestion
- The [Nightmare Zone plugin](https://github.com/runelite/runelite/wiki/Nightmare-Zone) can
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  now send notifications for your Nightmare zone overload boosts expiring prior to it expiring
```
Comment (Alexsuperfly) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:68]:
```suggestion
  now send notifications prior to your overload boosts expiring
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+![Weakened Zalcano with a hint arrow above her](/img/blog/1.6.25-Release/zalcano-hint-arrow.png)
+
+The [Inventory Tags plugin](https://github.com/runelite/runelite/wiki/Inventory-Tags) can now
+draw an underline as a more subtle option for your tagged items. Thanks to
+[@jualkaup](https://github.com/jualkaup) for this feature.
+
+![Inventory tag underlines](/img/blog/1.6.25-Release/inventory-tag-underline.png)
+
+We have added an anisotropic filtering setting to the [GPU
+plugin](https://github.com/runelite/runelite/wiki/GPU) which makes textures appear smoother and have
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:20]:
```suggestion
plugin](https://github.com/runelite/runelite/wiki/GPU) that makes textures appear smoother and have
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+fewer "jagged" and sharp pixels, especially when zoomed out as is shown in the comparison table
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:21]:
```suggestion
fewer "jagged" and sharp pixels, especially when zoomed out, as shown in the comparison table
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+| ![GPU plugin with anisotropic filtering enabled](/img/blog/1.6.25-Release/gpu-anisotropic-enabled.mp4) | ![GPU plugin with anisotropic filtering disabled](/img/blog/1.6.25-Release/gpu-anisotropic-disabled.mp4) |
+
+The [Item Stats plugin](https://github.com/runelite/runelite/wiki/Item-Stats) now displays a tooltip
+for duration effects granted by antipoison potions, anti-fire potions, and the like. Thanks to
+[@MMagicala](https://github.com/MMagicala) for this enhancement.
+
+![Potion duration tooltips](/img/blog/1.6.25-Release/potion-duration-tooltips.png)
+
+The [Mining plugin](https://github.com/runelite/runelite/wiki/Mining) now displays a session overlay
+which counts your ores mined, and ores mined per hour, along with an indicator of whether you are
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:36]:
```suggestion
that counts ores mined per hour and throughout the session, along with an indicator of whether you are
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+currently mining. Thanks to [@JZomerlei](https://github.com/JZomerlei) for this addition.
+
+![Mining session overlay while actively mining](/img/blog/1.6.25-Release/mining-session.png)
+![Mining session overlay when not actively mining](/img/blog/1.6.25-Release/mining-session-not-mining.png)
+
+There are also several smaller improvements and bug fixes, including:
+
+- The [Timers plugin](https://github.com/runelite/runelite/wiki/Timers) now shows an infobox with
+  your elapsed fight cave or inferno time
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:45]:
```suggestion
  your elapsed Fight Cave or Inferno time
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+- Tooltip labels have been added to clue scroll and minigame entries in the [HiScore
+  plugin](https://github.com/runelite/runelite/wiki/HiScore)
+- The [Agility plugin](https://github.com/runelite/runelite/wiki/Agility) now estimates your laps
+  per hour more accurately
+- Some Godwars Dungeon obstacles and the Prifddinas portals are now highlighted properly by the
+  [Agility plugin](https://github.com/runelite/runelite/wiki/Agility)
+- The [Clue Scroll plugin](https://github.com/runelite/runelite/wiki/Clue-Scroll) now shows a
+  location description for map clues
+- The [Cannon plugin](https://github.com/runelite/runelite/wiki/Cannon) now suggests a spot for
+  Meiyerditch Laboratory Bloodvelds
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:61]:
```suggestion
  Bloodveld in the Meiyerditch Laboratories
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+- You can now look up your Nightmare kill count with the [Chat Commands
+  plugin](https://github.com/runelite/runelite/wiki/Chat-Commands) using the `nm`, `tnm`, and
+  `nmare` abbreviations
+- Teleport tablets are now labeled by the [Item Identification
+  plugin](https://github.com/runelite/runelite/wiki/Item-Identification)
+- The [Nightmare Zone plugin](https://github.com/runelite/runelite/wiki/Nightmare-Zone) can now send
+  notifications prior to your overload boosts expiring
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:68]:
```suggestion
  notifications prior to your Overload boost's expiration
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 10]
```diff
@@ -0,0 +1,243 @@
...
+- The [Cooking plugin](https://github.com/runelite/runelite/wiki/Cooking) now updates its session
+  counter when drying sinew
+- The [Prayer plugin](https://github.com/runelite/runelite/wiki/Prayer) now shows a sensible
+  duration text for durations greater than 1 hour
+- Iorwerth Warriors are now understood to be elves by the [Slayer
+  plugin](https://github.com/runelite/runelite/wiki/Slayer)
+- XD and pleading emojis have been added to the [Emojis
+  plugin](https://github.com/runelite/runelite/wiki/Emojis) with the `Xd` and `(n_n)` triggers,
+  respectively
+- Vyres have been added to the [thieving skill
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:82]:
```suggestion
- Vyres have been added to the [Thieving skill
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md [hunk 11]
```diff
@@ -0,0 +1,243 @@
...
+  calculator](https://github.com/runelite/runelite/wiki/Skill-Calculator)
+- A bug where Chambers of Xeric personal best times were not recorded for certain team sizes has
+  been fixed
+- The [World Map plugin](https://github.com/runelite/runelite/wiki/World-Map) no longer shows a
+  misplaced quest icon for the Misthalin Mystery quest
+- Bird nests, and openings of the grubby, stone, Dorgesh-Kaan, and H.A.M chests are now tracked by
```
Comment (Broooklyn) [inline src/_posts/2020-08-26-10-00-1.6.25-Release.md:88]:
```suggestion
- Bird nests and the opening of grubby, stone, Dorgesh-Kaan, and H.A.M. chests are now tracked by
```

File: src/_posts/2020-08-26-10-00-1.6.25-Release.md (hunk 9 above)
Comment (Nightfirecat) [reply src/_posts/2020-08-26-10-00-1.6.25-Release.md:68]: This one is fine as-is

File: src/routes/blog.scss [hunk 12]
```diff
@@ -24,3 +24,7 @@
 #blog-list h5 {
   color: white;
 }
+
+video {
```
Comment (deathbeam) [inline src/routes/blog.scss:28]: This should be in app.scss as this is core component, and also it should be `max-width` instead of `width`

**Review by deathbeam — CHANGES_REQUESTED** [review]

**Review by deathbeam — APPROVED** [review]

**Pull request merged**